src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -413,6 +415,15 @@ function reducer(draft, action) {
       // TODO: This could break if a reusable definition is used multiple times in a form
       const step = getFormStep(formDefinition, draft.formSteps, true);
 
+      if (mutationType === 'changed' && schema['of-translations']) {
```

Comment:
I'm not sure about this part, because currently this is only to keep track of componentTranslations so they can be shown in the form builder in case validation fails. The actual translations that are sent to the backend are retrieved here: https://github.com/open-formulieren/open-forms/pull/2437/files#diff-88ee85ef9cb073c1e9150e72a526cb11c736468981c6361fa14c2f7bca22c24fR42 (from the formio config `of-translations`)

Reply:
This will be tackled in https://github.com/open-formulieren/open-forms/issues/2478